contrib/zmq/zmq_sub.py

```diff
@@ -69,6 +68,12 @@ async def handle(self) :
         elif topic == b"rawtx":
             print('- RAW TX ('+sequence+') -')
             print(binascii.hexlify(body))
+        elif topic == b"sequence":
+            hash = body[:32].hex()
+            label = chr(body[32])
+            mempool_sequence = None if len(body) != 32+1+8 else struct.unpack("<Q", body[32+1:])[0]
+            print('- SEQUENCE ('+sequence+') -')
+            print("{} {} {}".format(hash, label, mempool_sequence))
```

Comment:
Format looks unnecessary here :)
```suggestion
            print(hash, label, mempool_sequence)
```